extension/content.js

```diff
@@ -170,6 +170,15 @@ function addPatchDiffLinks() {
 	`);
 }
 
+function removeDiffSigns() {
+	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
+		const $element = $(element);
+		const html = $element.html();
+
+		$element.html(html.replace(/^[+-]/, ' '));
```

Comment:
No, I wasn't clear enough. You just do the `.replace` on the text node instead of all the HTML. Not remove the text node.